test/integration/features/cmd-hostfolder.feature

```diff
@@ -0,0 +1,50 @@
+@cmd-hostfolder @command
+
+Feature: hostfolder commands
+  As a user I can perform basic operations of Minishift with hostfolder mount
+
+  Scenario: User should not be able to list hostfolder when it does not exist
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift hostfolder list" succeeds
+     Then stdout should contain
+     """
+     no host folders defined
+     """
+  Scenario: User should be able to add a hostfolder of type SSHFS
+    When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+    When executing "minishift hostfolder add -t sshfs --source /home/minishift_ci --target /mnt/sda1/myshare myshare" succeeds
```

Comment:
@agajdosi Agree.  Also this test should be able to run on Windows, macOS.

Reply:
Since this should run in any OS, it is better to refrain from using an absolute path. We should be able to use a variable `HOME`, and this is possible to setting a variable into the template.